check_correctness/README.md

```diff
@@ -0,0 +1,28 @@
+# Evaluate material model on mesh
+
+## Compilation
+
+```
+mkdir build && cd build
+cmake ..
+make
+```
+
+## Usage
+
+```
+mpirun -n 4 ./check_correctnes -m mesh.h5
+```
+
+## Description?
+
+1. Read `mesh.h5`
+2. For each element check that:
+  - if a face is a regular or a dynamic rupture face, the neighbor *has to exist*.
+  - if a face is a free-surface or an absorbing face, the neighbor *must not exist*.
+3. The program aborts, when the first face is found that breaks the rules above.
```

Comment:
The program aborts when the first face that breaks the above rules is found.?

Reply:
No, it runs until the end and then exits with an error code.

Reply:
ok then 
"The program aborts, when one or several faces are found breaking the rules above."